assets/javascripts/discourse/templates/modal/create-channel.hbs

```diff
@@ -8,15 +8,11 @@
     options=(hash none="chat.create_channel.choose_category.none")
   }}
 
-  <div class="create-channel-hint">
-    {{html-safe
-      (i18n
-        "chat.create_channel.choose_category.hint"
-        link=categoryHint.link
-        category=categoryHint.category
-      )
-    }}
-  </div>
+  {{#if categoryHintText}}
```

Comment:
now that we name it permissions in API, we should maybe reflect the change here?